actors-akka/src/main/scala/reactivemongo/actors/util/package.scala

```diff
@@ -0,0 +1,15 @@
+package reactivemongo.actors
+
+import akka.util.{
+  ByteString => PekkoByteString,
```

Comment:
Don't see why Akka type should be prefixed as `Pekko`

Reply:
Addressed in 87d3301898f6bbf86f7167a2c141cc2bbd9e8370